src/nemoWorker/NemoRunner.ts

```diff
@@ -218,6 +228,45 @@ export class NemoRunner {
     return this.program.getOutputPredicates();
   }
 
+  public async experimentalNewTracingTreeForTable(
+    tree_for_table_query: TreeForTableQuery,
+  ): Promise<TreeForTableResponse> {
+    return TreeForTableResponseFromJSON(
+      this.engine.experimentalNewTracingTreeForTable(
+        TreeForTableQueryToJSON(tree_for_table_query),
+      ),
+    );
+  }
+
+  public async experimentalNewTracingTableEntriesForTreeNodes(
+    table_entries_for_tree_nodes: TableEntriesForTreeNodesQuery,
+  ): Promise<TableEntriesForTreeNodesResponseInner[]> {
+    const response = this.engine.experimentalNewTracingTableEntriesForTreeNodes(
+      TableEntriesForTreeNodesQueryToJSON(table_entries_for_tree_nodes),
+    );
+    return response.map(TableEntriesForTreeNodesResponseInnerFromJSON);
+  }
+
+  public static experimentalNewTracingTreeForTableMock(
+    tree_for_table_query: TreeForTableQuery,
+  ): TreeForTableResponse {
+    return TreeForTableResponseFromJSON(
+      NemoEngine.experimentalNewTracingTreeForTableMock(
+        TreeForTableQueryToJSON(tree_for_table_query),
+      ),
+    );
+  }
+
+  public static experimentalNewTracingTableEntriesForTreeNodesMock(
+    table_entries_for_tree_nodes: TableEntriesForTreeNodesQuery,
+  ): TableEntriesForTreeNodesResponseInner[] {
+    const response =
+      NemoEngine.experimentalNewTracingTableEntriesForTreeNodesMock(
+        TableEntriesForTreeNodesQueryToJSON(table_entries_for_tree_nodes),
+      );
+    return response.map(TableEntriesForTreeNodesResponseInnerFromJSON);
+  }
```

Comment:
These functions should be renamed and the mocks should be removed; also on the nemo "backend" side.